frontend-react/src/utils/Analytics.ts

```diff
@@ -0,0 +1,20 @@
+import { IEventTelemetry } from "@microsoft/applicationinsights-web";
+
+import { getAppInsights } from "../TelemetryService";
+
+export enum EventName {
+    DAILY_DATA_TABLE_FILTER = "Daily Data | Table Filter",
+    SUBMISSIONS_TABLE_FILTER = "Submission History | Table Filter",
+}
+
+export const TrackAppInsightEvent = (
```

Comment:
Very small point, but function names in JavaScript conventionally start with lowercase letters: `trackAppInsightEvent`.